research/templates/research/browse.html

```diff
@@ -16,7 +16,7 @@
 {% block style %}
 <link rel="stylesheet" href="{{ url_for('research_bp.static', filename='css/research.css') }}">
 <link rel="stylesheet" href="{{ url_for('research_bp.static', filename='lib/datatables/css/datatables.min.css') }}">
-<link rel="stylesheet" href="{{ url_for('research_bp.static', filename='lib/font-awesome/css/font-awesome.css') }}">
+<link rel="stylesheet" href="{{ url_for('static', filename='lib/font-awesome-4.7.0/css/font-awesome.min.css') }}">

```

Comment:
This link is not needed any more, if I am not mistaken. It is already included in the base template:
https://github.com/UtrechtUniversity/yoda-portal/blob/d45bae2013a9f5bcfff8cc739f653fc2ff2d7a5a/general/templates/general/base.html#L12